### PR TITLE
SciPy17 tutorial

### PR DESCRIPTION
**Pull request by jlstevens**
Material to merge into master for the SciPy 2017 tutorial.

### CODE REVIEW

File: notebooks/00-welcome.ipynb [hunk 1]
```diff
@@ -0,0 +1,131 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href='http://www.holoviews.org'><img src=\"assets/header_logo.png\" alt=\"HoloViews logo\" width=\"20%;\" align=\"left\"/></a>\n",
+    "<div style=\"float:right;\"><h2>00. Introduction and Setup</h2></div>"
```
Comment (jbednar) [inline notebooks/00-welcome.ipynb:8]: Should put the Bokeh logo here too, and maybe Datashader.

Comment (jlstevens) [reply notebooks/00-welcome.ipynb:8]: Sure.

File: notebooks/01-introduction-to-elements.ipynb [hunk 2]
```diff
@@ -0,0 +1,584 @@
...
+    "In this notebook we will explore a number of different element types and examine some of the ways that elements can supplement the supplied data with useful semantic data. There are many element types you might chose in the exercises and to help you make your choice, you can browse the [visual reference](http://holoviews.org/reference/index.html) which shows all the element types."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating elements\n",
+    "\n",
+    "All basic elements accept their data as a single, mandatory positional argument which may be supplied in a number of different formats, some of which we will now examine. A handful of *annotation* elements are exceptions to this rule, namely ``Arrow``, ``Text``, ``Bounds``, ``Box`` and ``Ellipse`` as they require additional positional arguments.\n",
```
Comment (jbednar) [inline notebooks/01-introduction-to-elements.ipynb:56]: Probably a simpler way to say this is something like:

    All elements (apart from Annotations) accept a single, mandatory positional 
    argument -- "the data".  Various different types and formats of data are 
    accepted, as we will start to explore below.

File: notebooks/01-introduction-to-elements.ipynb [hunk 3]
```diff
@@ -0,0 +1,584 @@
...
+   "metadata": {},
+   "source": [
+    "### Annotating the curve"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So far the only thing that we have done with our two lists (``xs`` and ``ys``) is wrap it in different types of elements. There is more to annotating your data with suitable semantics that just declaring the type, for instance we might want to specify what the x-axis and y-axis actual correpond to. Perhaps this parabola is the trajectory of a thrown ball:"
```
Comment (jbednar) [inline notebooks/01-introduction-to-elements.ipynb:132]: actually correspond

(there are various other typos; won't fix those now.)

**Comment by jlstevens** [comment]
There are a few small sections that still need to be added along with more polish but I think we can now merge this.

**Pull request merged**